spec/menu_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'menu'
+
+describe Menu do
+
+  it 'allows to display dishes as a well formatted list' do
+    subject.add_dish("Prosciutto",12)
+    expect(subject.display_menu).to eq('pizza: £12, burger: £10, Prosciutto: £12')
+    end
```

Comment:
Lint/BlockAlignment: `end` at 8, 4 is not aligned with `it 'allows to display dishes as a well formatted list' do` at 5, 2.